test/gtest/ucp/ucp_test.cc

```diff
@@ -665,14 +660,30 @@ int ucp_test_base::entity::get_num_eps(int worker_index) const {
     return m_workers[worker_index].second.size();
 }
 
-size_t ucp_test_base::entity::get_rejected_cntr() const {
+void ucp_test_base::entity::add_err(ucs_status_t status) {
+    switch (status) {
+    case UCS_ERR_REJECTED:
+        ++m_rejected_cntr;
+        /* fall through */
+    default:
+        ++m_err_cntr;
+    }
+}
+
+size_t ucp_test_base::entity::get_err_num(ucs_status_t status) const {
+    EXPECT_EQ(UCS_ERR_REJECTED, status) << "Missed counter for "
+                                        << ucs_status_string(status)
+                                        << " type of errors";
```

Comment:
better move it to test case itself, not as part of a generic getter function

Reply:
what do you mean? this is a friendly message about not implemented functionality to test developer...
I thought `get_err_num(REJECT)` looks better (and easily extendable in additional) than get_reject_num, get_unreachable_num, etc

Reply:
i mean move this message to test itself, outside this function
this function will just return the number

Reply:
this message is needed to warn that entity doesn't count any specific errors excepting "reject" and it should be added on demand.
the message is meaningless outside this function since test requests only "reject" counter for now.
will try to reword it